Dockerfile

```diff
@@ -38,26 +38,13 @@ RUN         apt-get update \
             && docker-php-ext-configure zip \
             && docker-php-ext-install zip \
             && a2enmod rewrite \
-
-
-####################
-# Install Composer #
-####################
-
+# Install Composer
             && curl -sS https://getcomposer.org/installer | php \
             && mv composer.phar /usr/local/bin/composer \
-
-###############
-# Install NPM #
-###############
-
-            && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+# Install NPM
+            && curl -sL https://deb.nodesource.com/setup_18.x | bash - \
```

Comment:
I would stick LTS version, 18 is not LTS and some packages might not work.

Reply:
done